content/getting-started/guides/webhook-apps/index.mdx

```diff
@@ -157,7 +157,22 @@ Each webhook contains the following properties:
 - `token` and `license_id` – your authentication credentials that let you call LiveChat’s Configuration API v2 methods. You won’t need to use them unless you want to make a call to LiveChat’s API right when you receive a webhook. In that case, you just need to pass these `token` and `license_id` credentials in your API call.
 - additional information – please read the [Webhook data types](#webhook-data-types) section.
 
-When your server receives a webhook from LiveChat, it should respond with `HTTP 200` response. Otherwise, LiveChat will retry sending the webhook to your service for a number of times unless it receives the correct `HTTP 200` response.
+<Warning>
+
+Note: When your server receives a webhook from LiveChat, it should respond with **HTTP 200** response. Otherwise, LiveChat will retry sending the webhook to your service for a number of times unless it receives the correct **HTTP 200** response.
```

Comment:
```suggestion
When your server receives a webhook from LiveChat, it should respond with **HTTP 200** response. Otherwise, LiveChat will retry sending the webhook to your service a number of times until it receives the correct **HTTP 200** response.
```

Reply:
- We use the yellow component, so I don't think we need 'Note'.
- 'until' sounds better as we're discussing _time_